Update networking components docs (#978)
liqotech/liqo: docs/pages/concepts/networking/components/route.md

![Liqo Route Components](../../../../images/liqonet/liqo-route-components.png)

The Liqo-Route component is kept simple as much as possible by design. The reason is to have a solution that is agnostic to supported CNI plugins and cloud providers. It does not interfere with the existing configuration or existing network interfaces. It does that by creating a VXLAN network interface and using a custom routing table for routes needed by the cross cluster traffic.

{{% notice note %}}
One thing that the component does not is to mess with the `iptables` configuration to prevent the SNATing of the traffic leaving the cluster. Usually the cross cluster traffic is SNATed by the CNI plugins, and if this behaviour is not desired they provide ways to disable the masquerading.
{{% /notice %}}

#### Overlay Operator

> When the Liqo Route component starts it creates a VXLAN network interface named `liqo.vxlan`. The reference deployment for VXLAN is to use an IP multicast group to join the other VTEPs. Since we can not assume that multicast is supported everywhere, the VXLAN overlay is deployed in `unicast` with static L2 entries. The Overlay Operator is the glue that updates the `FDB` entries for the `liqo.vxlan` device. The logic is straightforward, the operator watches for the Liqo Route pods in the namespace where Liqo is deployed and does the following:

Reviewer: ```suggestion
When the Liqo Route component starts it creates a VXLAN network interface named `liqo.vxlan`. The reference configuration for VXLAN is to use an IP multicast group to join the other VTEPs. Since we cannot assume that multicast is supported everywhere, the VXLAN overlay is deployed in `unicast` mode with static L2 entries. The Overlay Operator is the glue that updates the `FDB` entries for the `liqo.vxlan` device. The logic is straightforward, the operator watches for the Liqo Route pods in the namespace where Liqo is deployed and does the following:
```
